packages/ember-data/lib/system/adapter.js

```diff
@@ -330,6 +330,14 @@ var Adapter = Ember.Object.extend({
 
     Serializes the record update and sends it to the server.
 
+    The updateRecord method is expected to return a promise that will
+    resolve with the serialized record. This allows the backend to
```

Comment:
That is correct. Maybe I'm using the wrong terminology. I think of the raw object before normalizing the model by serializer as the "serialized data" and the json object after it has been run through `normalizeResponse` to be the "normalized data".